tests/codebuild/create_tests.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+source run_test.sh
+source inject_tests.sh
+
+function run_canary_tests
```

Comment:
Awesome refactor! Can you add a short documentation blurb to this file or the functions themselves? (one thing that would help is explaining how these differ from `delete_tests.sh`)